src/components/ErrorMesage/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './ErrorMessage';
```

Comment:
There is a typo in the directory name. It should be 'ErrorMessage' instead of 'ErrorMesage'.